app/models/vacancy.rb

```diff
@@ -20,6 +19,14 @@ class Vacancy < ApplicationRecord
     'full_time' => 0
   }.merge(FLEXIBLE_WORKING_PATTERN_OPTIONS).freeze
 
+  JOB_SORTING_OPTIONS = [
```

Comment:
This could entirely be managed by I18n. You could do something like:
```
  jobs:
    sort_options:
      publish_on_desc: Newest job listing
      publish_on_asc: Oldest job listing
      expiry_time_desc: Least time to apply
      expiry_time_asc: Most time to apply

I18n.t('jobs.sort_options')
=> {:publish_on_desc=>"Newest job listing", :publish_on_asc=>"Oldest job listing", :expiry_time_desc=>"Least time to apply", :expiry_time_asc=>"Most time to apply"}
```

Reply:
Once the default `most_relevant` option is prepended and the arrays are mapped I'm not sure this ends up being simpler or clearer unfortunately